### ISSUE TITLE
Strange behavior of SlevomatCodingStandard.Operators.NegationOperatorSpacing

### ISSUE THREAD

**Issue by dshemin** [slevomat/coding-standard#917]
HI!
When I specify in my ruleset this rule with the next configuration:
```xml
<rule ref="SlevomatCodingStandard.Operators.NegationOperatorSpacing">
    <properties>
        <property name="spacesCount" value="0"/>
    </properties>
</rule>
```

I got next error messages after invoking `phpcs`:
```
---------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------
 83 | ERROR | [x] Expected exactly 0 space after "-", 0 found.
---------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
---------------------------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
@dshemin Please post code that throws this error.

**Comment by dshemin** [comment]
Unfortunately, I can not share the whole code with you but here a small part of the files for which I got errors:

```php
public function testDeleteNotExists(): void
{
    $this->actingAs($this->user, 'api')
        ->json(Request::METHOD_DELETE, 'api/v1/ips', [ 'ids' => [ -1, 0 ] ]) // I got error for this line
        ->assertStatus(Response::HTTP_UNPROCESSABLE_ENTITY)
        ->assertJsonStructure([
            'message',
            'errors' => [
                'ids.0',
                'ids.1',
            ]
        ]);
}
```

```php
public function constructorNegativeDataProvider(): array
{
    return [
        'invalid total number' => [
            -1, // I got error for this line
            2,
            'Total value should be greater or equal to 0.',
        ],
        'invalid free number' => [
            1,
            -2, // I got error for this line
            'Free value should be greater or equal to 0.',
        ],
    ];
}
```

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/d0833ea787f87ea667a84dc8c3745bbf84277009

As a workaround just remove this from your configuration:

```
    <properties>
        <property name="spacesCount" value="0"/>
    </properties>
```

Zero is default value.
